index.html

```diff
@@ -10,33 +10,38 @@
   <header>
 	<label><a href="#" class="logo">My Logo</a></label>
 	  <nav>
-	    <ul>
-		  <li><a href="#" class="bar">
+      <ul>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+	    <div class="bar">
 		    <div class="b1"></div>
 			<div class="b2"></div>
-			<div class="b3"></div></a>
-		  </li>
-		</ul>
+			<div class="b3"></div>
+		</div>
 	  </nav>	
   </header>
-	<main class="backimg">
-	  <section class="h">
+	<main>
+	  <section>
+      <div class="container">
 		 <h1>I’m Mariane<br>Glad to see<br>you!</h1>
 		   <p class="p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 			 <p class="p2">LET’S CONNECT</p>
-	        <ul class="images">
-			    <li><a href=""><img src="img/Vector.png" alt="icon1"></a></li>
-			    <li><a href=""><img src="img/Linkedin.png" alt="icon2"></a></li>
-			    <li><a href=""><img src="img/Vector(1).png" alt="icon3"></a></li>
-			    <li><a href=""><img src="img/Vector(2).png" alt="icon4"></a></li>
-			    <li><a href=""><img src="img/Vector(3).png" alt="icon5"></a></li>
+	        <ul class="icons">
+			    <li><a href=""><img src="img/Vector.png" alt="icon1" class="icon-item"></a></li>
+			    <li><a href=""><img src="img/Linkedin.png" alt="icon2" class="icon-item"></a></li>
+			    <li><a href=""><img src="img/Vector(1).png" alt="icon3" class="icon-item"></a></li>
+			    <li><a href=""><img src="img/Vector(2).png" alt="icon4" class="icon-item"></a></li>
+			    <li><a href=""><img src="img/Vector(3).png" alt="icon5" class="icon-item"></a></li>
 	       </ul>
+         </div>
 	    </section>
     </main>
-    <section class="continer">
+    <section>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 1-Main (1)](https://user-images.githubusercontent.com/69638013/165755259-db6c59c8-b886-4b30-8b60-47d7e81d0e7b.png) | ![Screenshot_95](https://user-images.githubusercontent.com/69638013/165755342-53ac89db-31cf-4a51-a88b-68e61c614eef.png)

- [ ] Please increase the width of each card to look more like the figma design 👍🏿 

- [ ] Kindly add button interactions for buttons in this section 👍🏿 
![Screenshot_94](https://user-images.githubusercontent.com/69638013/165753945-5403db37-e85d-4b64-a975-de4c5f32f3a8.png)